codeBlock-1984/fila
# Create file creation endpoint

#### What does this PR do?
It adds the endpoint for creating a new file.

### Type of change.

#### Description of task completed.
- Create file controller, service, repo and router.
- Install validator package, uuid. And implement file validation and creation.
- Install winston and setup error-logging.
- Create middleware handlers for app and routers.

#### How can this be tested?
- Having setup the project locally, `cd` into the project root direct and run `npm run serve`.
- Send the following request using CURL:
```
curl -d '{"name":"test file", "size":1024, "format": "PDF"}' -H "Content-Type: application/json" -X POST http://localhost:3600
```
- You should get a success response.


#### Relevant screenshots.
- N/A

### Associated Agile Story/Issue/Task.
- N/A
